terraform/eval_count.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/zclconf/go-cty/cty/gocty"
 )
 
-// evaluateResourceCountExpression is our standard mechanism for interpreting an
+// evaluateCountExpression is our standard mechanism for interpreting an
```

Comment:
You may want to update the comment so it's less resource-specific!

Reply:
Thanks! I did! it was just in the other PR :(